src/compute-types/src/dyncfgs.rs

```diff
@@ -34,7 +34,7 @@ pub const LINEAR_JOIN_YIELDING: Config<&str> = Config::new(
 /// Enable lgalloc for columnation.
 pub const ENABLE_COLUMNATION_LGALLOC: Config<bool> = Config::new(
     "enable_columnation_lgalloc",
-    false,
+    true,
```

Comment:
Makes sense to me but we should have @antiguru weigh in.

Reply:
LGTM

Reply:
@antiguru: how does this work in practice? Does `lgalloc` on Docker require attaching a volume to the container?

Reply:
It "just works." The image has write access to the overlay, so it'll just use that for its storage. Not ideal, but we could teach users how to mount a different directory for scratch storage

Reply:
Since this is probably hard to use correctly, and easy to misuse and go out of disk, we'll opt to disable for now.